src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -232,8 +236,7 @@ public void setUp() throws IOException, COSVisitorException {
     @After
     public void tearDown() {
         if (!pdfFile.delete()) {
-            System.err
-                    .println("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
+            LOGGER.info("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
```

Comment:
I'll replace it by [deleteOnExit](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#deleteOnExit%28%29).